docs/usage/shoot/shoot-workload-identity.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Shoot Workload Identity
+description: Configure access to infrastructure accounts via workload identity instead of static credentials
+---
+
+# Shoot Workload Identity
+
+[`WorkloadIdentity`](../../api-reference/security.md#workloadidentity) is a resource that allows workloads to be presented before external systems by giving them identities managed by Gardener.
+As `WorkloadIdentity`s do not directly contain credentials we gain the ability to create `Shoot`s without the need of preliminary exchange of credentials.
+For that to work users should establish trust to the Gardener Workload Identity Issuer in advance.
+The issuer URL can be read from the [Gardener Info ConfigMap](../gardener/gardener_info_configmap.md).
+
+> [!TIP]
+> `Shoot`s that were previously using `Secret`s as authentication method can also be migrated to use `WorkloadIdentity`.
```

Comment:
Related to my previous comment - how this could happen? By updating the CredentialsBinding or creating a new one?